### PR TITLE
[DowngradePhp71] Add DowngradeIsIterableRector

### PR DESCRIPTION
**Pull request by samsonasik**
To handle prefixed undefined function is_iterable notice:

Screen Shot 2021-04-22 at 19 18 13

### CODE REVIEW

File: rules/DowngradePhp71/Rector/FuncCall/DowngradeIsIterableRector.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+     * @param FuncCall $node
+     */
+    public function refactor(Node $node): ?Node
+    {
+        /** @var mixed $arg */
+        $arg = $node->args[0]->value;
+        $funcCall = $this->nodeFactory->createFuncCall('is_array', [$arg]);
+        $instanceOf = new Instanceof_($arg, new FullyQualified('Traversable'));
+
+        return new BooleanOr($funcCall, $instanceOf);
```
Comment (samsonasik) [inline rules/DowngradePhp71/Rector/FuncCall/DowngradeIsIterableRector.php:69]: @TomasVotruba direct return `BooleanOr` seems cause hang in tests, do you have idea how to check it? it working ok on `print_node` check.

Comment (samsonasik) [reply rules/DowngradePhp71/Rector/FuncCall/DowngradeIsIterableRector.php]: fixed with `traverseNodesWithCallable()` 🎉

Comment (samsonasik) [reply rules/DowngradePhp71/Rector/FuncCall/DowngradeIsIterableRector.php:74]: I just re-check that it need `isName()` check first to skip other function call

**Comment by TomasVotruba** [comment]
:+1:

**Pull request merged**